[2024-06-28T03:23:19.538+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T03:23:19.734+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.fetch_weather_data scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-06-28T03:23:19.779+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.fetch_weather_data scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-06-28T03:23:19.780+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T03:23:19.857+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2024-06-27 00:00:00+00:00
[2024-06-28T03:23:19.903+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=371) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-28T03:23:19.919+0000] {standard_task_runner.py:63} INFO - Started process 382 to run task
[2024-06-28T03:23:19.923+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'fetch_weather_data', 'scheduled__2024-06-27T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpyvqu3c2b']
[2024-06-28T03:23:19.945+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask fetch_weather_data
[2024-06-28T03:23:20.197+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_pipeline.fetch_weather_data scheduled__2024-06-27T00:00:00+00:00 [running]> on host 29974a26e5b4
[2024-06-28T03:23:20.618+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:00:00+00:00'
[2024-06-28T03:23:20.620+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T03:23:21.009+0000] {python.py:237} INFO - Done. Returned value was: ./data/weather_data_paris_20240628032320.csv
[2024-06-28T03:23:21.009+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T03:23:21.122+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=fetch_weather_data, run_id=scheduled__2024-06-27T00:00:00+00:00, execution_date=20240627T000000, start_date=20240628T032319, end_date=20240628T032321
[2024-06-28T03:23:21.265+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T03:23:21.348+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-28T03:23:21.367+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-28T03:25:07.470+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T03:25:07.512+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.fetch_weather_data scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-06-28T03:25:07.523+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.fetch_weather_data scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-06-28T03:25:07.523+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T03:25:07.542+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2024-06-27 00:00:00+00:00
[2024-06-28T03:25:07.551+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=448) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-28T03:25:07.554+0000] {standard_task_runner.py:63} INFO - Started process 450 to run task
[2024-06-28T03:25:07.555+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'fetch_weather_data', 'scheduled__2024-06-27T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpnizq8rtc']
[2024-06-28T03:25:07.557+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask fetch_weather_data
[2024-06-28T03:25:07.652+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_pipeline.fetch_weather_data scheduled__2024-06-27T00:00:00+00:00 [running]> on host 29974a26e5b4
[2024-06-28T03:25:07.874+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:00:00+00:00'
[2024-06-28T03:25:07.875+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T03:25:07.918+0000] {python.py:237} INFO - Done. Returned value was: ./data/weather_data_paris_20240628032507.csv
[2024-06-28T03:25:07.919+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T03:25:07.955+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=fetch_weather_data, run_id=scheduled__2024-06-27T00:00:00+00:00, execution_date=20240627T000000, start_date=20240628T032507, end_date=20240628T032507
[2024-06-28T03:25:08.013+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T03:25:08.085+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-28T03:25:08.092+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-28T03:30:00.968+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-28T03:30:01.017+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.fetch_weather_data scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-06-28T03:30:01.032+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.fetch_weather_data scheduled__2024-06-27T00:00:00+00:00 [queued]>
[2024-06-28T03:30:01.033+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-28T03:30:01.057+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2024-06-27 00:00:00+00:00
[2024-06-28T03:30:01.070+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=583) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-28T03:30:01.073+0000] {standard_task_runner.py:63} INFO - Started process 585 to run task
[2024-06-28T03:30:01.074+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_data_pipeline', 'fetch_weather_data', 'scheduled__2024-06-27T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmple_b4xm6']
[2024-06-28T03:30:01.076+0000] {standard_task_runner.py:91} INFO - Job 17: Subtask fetch_weather_data
[2024-06-28T03:30:01.154+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_data_pipeline.fetch_weather_data scheduled__2024-06-27T00:00:00+00:00 [running]> on host 29974a26e5b4
[2024-06-28T03:30:01.348+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='fetch_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T00:00:00+00:00'
[2024-06-28T03:30:01.350+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-28T03:30:01.395+0000] {python.py:237} INFO - Done. Returned value was: ./data/weather_data_paris_20240628033001.csv
[2024-06-28T03:30:01.395+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-28T03:30:01.439+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=fetch_weather_data, run_id=scheduled__2024-06-27T00:00:00+00:00, execution_date=20240627T000000, start_date=20240628T033001, end_date=20240628T033001
[2024-06-28T03:30:01.494+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-28T03:30:01.539+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-28T03:30:01.542+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
